src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -121,6 +121,9 @@ internal class RazorLanguageServerClient : ILanguageClient, ILanguageClientCusto
 
         public async Task<Connection> ActivateAsync(CancellationToken token)
         {
+            // Swap to background thread, nothing below needs to be done on the UI thread.
+            await TaskScheduler.Default;
```

Comment:
I didn't know you could swap to a background thread this way, neat 😮

Reply:
I assume this is basically a no-op if you're already on the background?

Reply:
> I assume this is basically a no-op if you're already on the background?

Yup!